independent-publisher-connectors/Secure Code Warrior/apiDefinition.swagger.json

```diff
@@ -0,0 +1,7075 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Secure Code Warrior",
+    "description": "This [Secure Code Warrior](https://securecodewarrior.com) connector provides programmatic access to the learning platform. Secure Code Warrior makes secure coding a positive and engaging experience for developers as they increase their software security skills. With our flagship Learning Platform, we guide each coder along their own preferred learning pathway, so that security-skilled developers become the everyday superheroes of our connected world.",

```

Comment:
@troystaylor The URL formatting may not work in all the Power Automate pages. So please provide the description some thing like this:

`"description": "This Secure Code Warrior (https://securecodewarrior.com) connector .."`

Reply:
Done